docs/Selectors.md

```diff
@@ -147,6 +131,9 @@ You can use xPath to also traverse up and down the DOM tree, e.g.
 const parent = paragraph.$('..');
 console.log(parent.getTagName()); // outputs: "body"
 ```
+## id
+
+Finding element by id has no specific syntax in WebDriver and one should use either CSS selectors (`#<my element ID>`) or xPath (`//*[@id="<my element ID>"]`).
```

Comment:
```suggestion
Finding element by id has no specific syntax in WebDriver and one should use either CSS selectors (`#<my element ID>`) or xPath (`//*[@id="<my element ID>"]`). However some drivers (e.g. [Appium You.i Engine Driver](https://github.com/YOU-i-Labs/appium-youiengine-driver#selector-strategies)) might still [support](https://github.com/YOU-i-Labs/appium-youiengine-driver#selector-strategies) this selector.
```

Reply:
Sorry Christian. Did not understand that you were waiting for me and thought the ping was to initiate a french translation (for which I'm neither qualified nor available). As you may have noticed, I'm new to WebdriverIO, Web development & GitHub... So I fully trust you for you amendment !